### PR TITLE
Update AlphaAnimals_CE_Patch_Race_MycoidColossus.xml

### PR DESCRIPTION
**Pull request by Breadbox2k19**

### CODE REVIEW

File: Alpha Animals/ThingDefs_Races/AlphaAnimals_CE_Patch_Race_MycoidColossus.xml [hunk 1]
```diff
@@ -32,29 +39,28 @@
 					<tools>
 						<li Class="CombatExtended.ToolCE">
 							<capacities>
-								<li>Bite</li>
+								<li>BiteBlunt</li>
```
Comment (Juanfrank) [inline Alpha Animals/ThingDefs_Races/AlphaAnimals_CE_Patch_Race_MycoidColossus.xml:42]: CE only accepts bite, cut, blunt and poke afaik. Changing the capacity to something else might break it. If you wanna have both you have to do 
`<li>Bite</li>`
`<li>Blunt</li>`

Either way the mechanic of bite includes a blunt and sharp component so it's redundant.

**Review by Juanfrank — CHANGES_REQUESTED** [review]

File: Alpha Animals/ThingDefs_Races/AlphaAnimals_CE_Patch_Race_MycoidColossus.xml (hunk 1 above)
Comment (Breadbox2k19) [reply Alpha Animals/ThingDefs_Races/AlphaAnimals_CE_Patch_Race_MycoidColossus.xml:42]: Biteblunt is a legit damage capacity accepted by CE, alot of vanilla animals in CE are patched with this. Biteblunt is simply blunt damage with a different label, it different from sharp damage with blunt penetration.

File: Alpha Animals/ThingDefs_Races/AlphaAnimals_CE_Patch_Race_MycoidColossus.xml [hunk 2]
```diff
@@ -32,29 +39,28 @@
...
 							</capacities>
-							<power>15</power>
-							<cooldownTime>1.65</cooldownTime>
+							<power>36</power>
+							<cooldownTime>3.3</cooldownTime>
 							<linkedBodyPartsGroup>HeadAttackTool</linkedBodyPartsGroup>
-							<armorPenetrationBlunt>1.144</armorPenetrationBlunt>
-							<chanceFactor>0.2</chanceFactor>
+							<armorPenetrationBlunt>15</armorPenetrationBlunt>
+							<chanceFactor>1</chanceFactor>
```
Comment (Juanfrank) [inline Alpha Animals/ThingDefs_Races/AlphaAnimals_CE_Patch_Race_MycoidColossus.xml]: What does this node do?

Comment (Breadbox2k19) [reply Alpha Animals/ThingDefs_Races/AlphaAnimals_CE_Patch_Race_MycoidColossus.xml]: Chance factor affects how likely the attack is chosen.

Comment (Juanfrank) [reply Alpha Animals/ThingDefs_Races/AlphaAnimals_CE_Patch_Race_MycoidColossus.xml]: 1 is 100% right so it would use it every single time

Comment (Breadbox2k19) [reply Alpha Animals/ThingDefs_Races/AlphaAnimals_CE_Patch_Race_MycoidColossus.xml]: No I don’t think it works like that, it just means it works as per normal as if it is not there.

Comment (Juanfrank) [reply Alpha Animals/ThingDefs_Races/AlphaAnimals_CE_Patch_Race_MycoidColossus.xml]: Then I'll just remove it since it's extra code

**Pull request merged**